### PR TITLE
feat: 3D profile 2.0

### PR DESCRIPTION
**Pull request by EllaMartirosyan**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| New feature     | ✔                                                                        |

### CODE REVIEW

File: src/AUTOGENERATED/GraphQLClass.ts [hunk 1]
```diff
@@ -570,15 +564,17 @@ export class Layer3DRecord {
...
     @Field({ nullable: true })
     public maxFlightAlt?: number;
     @Field({ nullable: true })
     public geographicArea?: string;
     @Field({ nullable: true })
     public productBoundingBox?: string;
+    @Field({ nullable: true })
+    public productSource?: string;
+    @Field({ nullable: true })
+    public productStatus?: string;
```
Comment (alebinson) [inline src/AUTOGENERATED/GraphQLClass.ts:577]: Should be **not-Nullable and enum**

**Review by alebinson — CHANGES_REQUESTED** [review]
see my comments

File: src/AUTOGENERATED/GraphQLClass.ts (hunk 1 above)
Comment (EllaMartirosyan) [reply src/AUTOGENERATED/GraphQLClass.ts]: Fixed in mc-models

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**